src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
+
+    add(operand) {
+      calculator.result += operand;
```

Comment:
nice, i have 'this.result' (it did not work)))) 'calculator.result' is good idea, but what if we need to change object's name?

Reply:
The object calculator is wrapped in the makeCalculator() function. The purpose of the function is to use it "as is", without changing anything in it.
It’s a pity that JS doesn’t have encapsulation as such and access modifiers that could be used to forcibly close access to it.

Assigning any name to the function return (const calculator = makeCalculator(); const myCalculator = makeCalculator(); const someCalculator = makeCalculator(); ...) will not break the function.

Reply:
it won't, but you surely need to use this in your object methods